### PR TITLE
[DNM UNTIL POSITIVE MOODIES ADDED] Seeing people die and pets die upsets you

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Seeing people or pets die, giving a negative moodlet. Person grief and pet grief stacks. Most antagonists are immune to people grief, but not pet grief. People will grieve for Ian, Runtime, Wags-His-Tail and Renault. People will only be upset seeing a human die if that human was player controlled at the time of its death.
del: Antagonists no longer have boosts to their mood, due to their grim nature.
/:cl:

- Only /mob/living/carbon/human and certain pets have the "grief" component.

- Antag datums now have a `mind_traits` list var which contains `list(TRAIT_GRIM)` by default. Some of the more non-violent antagonist datums are equally affected by witnessing death. They are Abductors, non-deathsquad ERT, Valentines, Greentexters and Santa.

- Yes, you probably could make a Grim quirk that gives TRAIT_GRIM, but someone else can do that.

- Wags-His-Tail is now a special subtype

### CODE REVIEW

**Comment by XDTM** [comment]
minor nerf to validhunting 👍

**Comment by nicbn** [comment]
Killing a corgi as nonantag should make your character insane, just as you are

**Comment by ExcessiveUseOfCobblestone** [comment]
Since the moodlet antags get was specifically for this, you should remove it.

**Comment by k-k-karma** [comment]
this is dumb and nerfs validhunting

**Comment by k-k-karma** [comment]
oh and griffabductors

**Comment by DaxDupont** [comment]
Bad pr this will playing medbay hell.

That or exclude medical staff since doctors irl go through desensitivity training for a reason.

**Comment by subject217** [comment]
go back to hippie lol

**Comment by Bombpants** [comment]
Maybe make certain professions, such as SEC, med bay staff, and Chappy less affected by death? All three deal with the dead and dying.

**Comment by coiax** [comment]
Quirks already exist that make mood less affecting of you, and if anything, this will mean that Medbay will have to make frequent trips to the bar for booze to block out the pain.

Just like real doctors!

**Comment by RogueSteampunker** [comment]
I don't really think making jobs like medical and sec even more hell for the sake of a silly debuff that'll get touted as "a serious deterrent to validhunting" is a good idea

**Comment by tuypo1** [comment]
Awful idea, death is a fact of life on space station 13.

**Comment by DaxDupont** [comment]
The whole u can use quirks to be less affected is an awful excuse.

Playing doctor is often already suffering especially when your a team of one or two and you're only making those situations worse and you push people away with it while indirectly buffing antags.

**Comment by DaxDupont** [comment]
Also sure have that single doctor having to go to the bar leaving medbay unattended is a great fucking idea.

**Comment by ExcessiveUseOfCobblestone** [comment]
Doctors/Security/Roboticists should certainly be less affected by this than other jobs since they regularly deal with dead bodies (but it's still sad to see john doe ur coworker die :sad:)

**Comment by tuypo1** [comment]
Fuck no, john doe was an arsehole.

**Comment by subject217** [comment]
Positive moodlet for successfully defibrillating someone
Positive moodlet for making a new cyborg
Positive moodlet for using the telegulag

**Comment by bigfatbananacyclops** [comment]
we need a pyschopath:sociopath quirk that seeing people die makes you happy

**Comment by bigfatbananacyclops** [comment]
@daxdupont thats kind of how it would work IRL, wouldnt you be losing your mind seeing 40 people dead in a hospital. Its just how it works

**Comment by optimumtact** [comment]
I think we need a lot more ways to generate yourself positive moodlets before we can do this

**Comment by 4dplanner** [comment]
Having grim as a quirk option is not an acceptable fix for the doctor issue as quirks are picked prior to profession roll. If I roll for atmos tech but set medical as my second choice I shouldn't have to choose between having a useless quirk for an atmos tech or lacking a vital quirk for  doctor.

**Comment by subject217** [comment]
I gave you some cool examples for positive moodlets to add to counteract this

What if we add a Psychologist role who can give special hugs that give extra moodlet bonus? I'd play it

**Comment by tuypo1** [comment]
>Positive moodlet for using the telegulag
Better idea, positive Moodlet for using the reeducation chamber.
Of course I imagine that would be a programming nightmare.

**Comment by DaxDupont** [comment]
i taisnkg its a fqfuckin daumg idaea k cose tis erpr

**Comment by 4dplanner** [comment]
> i taisnkg its a fqfuckin daumg idaea k cose tis erpr

Yes

**Comment by Tlaltecuhtli** [comment]
lazarus injecting the pet should clean up the buff and make you believe in jesus

**Comment by necromanceranne** [comment]
The horror...the horror...

**Comment by Grazyn** [comment]
I think it's time for that antidepressant drug that was promised in the original moodlet PR about never delivered

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -142,6 +142,16 @@
 	description = "<span class='boldwarning'>I recently saw my own corpse...</span>\n"
 	mood_change = -6
 
+/datum/mood_event/someone_died
+	description = "<span class='warning'>I saw someone die, and it has affected me deeply.</span>\n"
+	mood_change = -6
+	timeout = 10 MINUTES
```
Comment (Mickyan) [inline code/datums/mood_events/generic_negative_events.dm]: There's an argument to be made about other moodlets expiring way too fast but that needs to change first, with a 10 minutes duration this is going to require constant attention to counter and if you're playing medical or shit is hitting the fan, you're going to have to either be insane or constantly drunk

**Comment by kingoftheautism** [comment]
murdering Ian is a war crime that makes you valid, no excuses

actually xenophobe trait that makes you happy instead if you see xenos dying is may or may not be interesting idea

good PR, nerfes those bastards who makes corgi meat burgers

**Comment by Qustinnus** [comment]
moodlets don't stack so medbay isn't that affected by this. However I feel like giving medical players TRAIT_GRIM is fair because doctors are often more acclimated to encountering death.

**Comment by FloranOtten** [comment]
> @DaxDupont thats kind of how it would work IRL, wouldnt you be losing your mind seeing 40 people dead in a hospital. Its just how it works. Psychological thing and it makes sense, maybe doctors should come with a special quirk that they were trained to hold well with deaths, but anyways i think the apathetic quirk should make you inmune to this effect @coiax

I mean, realism doesn't make it fun. In reality, going a (ingame) day without food or sleep would make you quite sad but having to sod off to do the dishes while your character sleeps isn't very fun, either.

**Comment by Durkel** [comment]
If kor was still with us he would tell you this is a bad pr and you shouldn't tell players how they should feel at any given moment.

**Comment by subject217** [comment]
we know kor didn't like moodlets, how kor felt about them is not particularly relevant considering /tg/ has ran with them and it's honestly a pretty compelling system that is still being improved and expanded all the time.

**Comment by Nickkoranda** [comment]
Good PR. Ask EMTs over their stress about watching people die in front of them (but don't actually).  But I do think saving people should negate/provide an massive mood buff.

File: code/datums/mood_events/generic_negative_events.dm (hunk 1 above)
Comment (Qustinnus) [reply code/datums/mood_events/generic_negative_events.dm]: I feel like I can reasonably make a PR soon that buffs times for moodlets.

**Comment by tuypo1** [comment]
>pretty compelling system
That's a pretty funny way of spelling fucking awful.

**Comment by subject217** [comment]
in your opinion. aren't you a hippie player or something?

**Comment by tuypo1** [comment]
Never touched hippy.
I was a /tg/ player years ago and recently came back

**Comment by Grazyn** [comment]
Why would you feel bad for killing a greyshirt who broke in your department?

**Comment by tuypo1** [comment]
You wouldn't feel bad at all.

File: code/modules/antagonists/_common/antag_datum.dm [hunk 2]
```diff
@@ -59,6 +60,18 @@ GLOBAL_LIST_EMPTY(antagonists)
 /datum/antagonist/proc/remove_innate_effects(mob/living/mob_override)
 	return
 
+// Handles application of traits to the mind, such as being jaded, or special abilities
+/datum/antagonist/proc/apply_mind_traits()
+	if(mind_traits)
```
Comment (Cyberboss) [inline code/modules/antagonists/_common/antag_datum.dm]: Line is unecessary

File: code/modules/antagonists/_common/antag_datum.dm [hunk 3]
```diff
@@ -59,6 +60,18 @@ GLOBAL_LIST_EMPTY(antagonists)
...
+		for(var/trait in mind_traits)
+			owner.add_trait(trait, "antagonist_[name]")
+
+// Handles removal of traits to the mind, such as being jaded, or special abilities
+/datum/antagonist/proc/remove_mind_traits()
+	if(mind_traits)
```
Comment (Cyberboss) [inline code/modules/antagonists/_common/antag_datum.dm]: Line is unecessary

File: code/modules/antagonists/ert/ert.dm [hunk 4]
```diff
@@ -12,6 +12,7 @@
 	var/list/name_source
 	show_in_antagpanel = FALSE
 	antag_moodlet = /datum/mood_event/focused
+	mind_traits = list()
```
Comment (Cyberboss) [inline code/modules/antagonists/ert/ert.dm]:
```suggestion
 	mind_traits = null
```

File: code/modules/antagonists/greentext/greentext.dm [hunk 5]
```diff
@@ -2,6 +2,7 @@
 	name = "winner"
 	show_in_antagpanel = FALSE
 	show_name_in_check_antagonists = TRUE //Not that it will be there for long
+	mind_traits = list()
```
Comment (Cyberboss) [inline code/modules/antagonists/greentext/greentext.dm]:
```suggestion
 	mind_traits = null
```

File: code/modules/antagonists/valentines/valentine.dm [hunk 6]
```diff
@@ -2,6 +2,7 @@
 	name = "valentine"
 	roundend_category = "valentines" //there's going to be a ton of them so put them in separate category
 	show_in_antagpanel = FALSE
+	mind_traits = list() // Make love, not war.
```
Comment (Cyberboss) [inline code/modules/antagonists/valentines/valentine.dm]:
```suggestion
 	mind_traits = null // Make love, not war.
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by anconfuzedrock** [comment]
IF you use the execution room properly you won’t have to be in it for when the person actually dies, be it by vacuum or chair or injection.
I like subject’s idea but I swear to fuck if telegulag gets any better wardens will be valid every round

**Comment by epochayur** [comment]
"just cancel it with quirks!"

bad PR

**Comment by ATH1909** [comment]
Death is a fact of life on SS13; this is a bad PR. Adding a grim quirk or trait for non-antags/doctors/roboticists is just a band-aid, really.

Will this change actually make the game more fun/enjoyable for anyone?

Keep the pet death moodlet debuff, but remove the human death debuff. Maybe the pet debuff should only apply to members of the dying pet's department (so the sloth would only demoralize members of Cargo, Runtime would only demoralize members of Medbay, etc.); otherwise, people will drag around pets while repeatedly killing and rezzing them (with, say, strange reagent) to demoralize large parts of the station.

**Comment by optimumtact** [comment]
you have to get bad mood and then maintain bad mood for a longer period of time to be made slow, also the hallucinations were removed.

bad moods are not hard to counter
get a hug, take a shower
eat a sweet food
go workout in the gym
you'll manage
the fact you can't run around as a grim mindless murderboner robot is a feature of this change
not a bug

**Comment by optimumtact** [comment]
@coiax death by brain removal surgery should not cause a mood by the way, or otherwise borging someone should give an offsetting mood.

**Comment by ATH1909** [comment]
> you have to get bad mood and then maintain bad mood for a longer period of time to be made slow, also the hallucinations were removed.
> 
> bad moods are not hard to counter
> get a hug, take a shower
> eat a sweet food
> go workout in the gym
> you'll manage
> the fact you can't run around as a grim mindless murderboner robot is a feature of this change
> not a bug

Because I'd much rather be doing those things than doing my gimmick/job.

This is a -6 debuff that lasts for 10 MINUTES. For reference, a hug is a +1 buff that lasts for about 2 minutes.

Again, I don't want to babysit my mood instead of doing whatever else I wanted to do (because let's face it, you can't do much when you have to drop out of whatever you're doing every 2 minutes to go to the other side of the station to boost your mood up again) for 10 minutes just because someone committed suicide at my HoPline/someone loses their internal organs and collapses in Medbay/I debrain someone as a roboticist (to his credit, though, oranges wants to fix that last one).

Again, this makes the game less fun overall for the sake of something that should just be RP'd if you actually want to deal with/interact with it, so why should it be added? What are we now, an MRP server?

**Comment by 4dplanner** [comment]
I'd be interested to see how this played out if other mood times were changed to be comparable

**Comment by 4dplanner** [comment]
Edited

**Comment by optimumtact** [comment]
Yes like I said, we need long lasting positive mood buffs for this to be acceptable.

**Comment by zxaber** [comment]
seeing a borg die should give double the bad mood (unless it gets rebooted).

**Comment by subject217** [comment]
seeing a borg die should give positive mood

**Comment by zxaber** [comment]
Thems fightin' words

**Comment by SouDescolado** [comment]
We rimworld now

**Comment by anconfuzedrock** [comment]
Heroin is good in vietnam, it can be good here.

**Comment by Shadowflame909** [comment]
If we do this, make the Clown and the Mime give positive moodlets for simply existing.

Also make them start with the Grim Quirk like antags, because the clown.

**Comment by Grazyn** [comment]
I don't like the idea of mime/clown abilities giving happy moods

Griefing mime keeps wall blocking your path > happy mood
You finally manage to catch up and kill him > sad mood

Yeah maybe let's not insist on telling people how their character should feel?

**Comment by subject217** [comment]
but what if instead of murdering someone over blocking your path you just like... beat them up or something

>be me le quality roleplayer as a security officer
>harmbaton le antag to death epic win
>le fail moodlets make me sad for watching someone die instead of happy for pwning antags

**Comment by Qustinnus** [comment]
adding something for pwning antags can be added although it could be used to meta, so rather add something for turning in evidence in some kind of machine.

**Comment by Qustinnus** [comment]
@Grazyn also why would we make the griefing abilities give good mood, just make the nice things good moods. iirc clowns can give good moods through hugging or something similar

**Comment by subject217** [comment]
I was being sarcastic. 

My point is that if you don't want to be sad, maybe you shouldn't kill them next time. *Oh no, God forbid death have any meaning in my videogame!* And one or two people dying shouldn't be enough to actually make someone insane as this PR currently stands, as I understand it.

**Comment by Shadowflame909** [comment]
^ War ops and Wizards

Cult, Revs, Traitors, Chang- Literally any antag with a chance of mass death and destruction. Most likely all of them, can cause the major bad moodlet. We need more ways for the major good moodlet

**Comment by Shardj** [comment]
> ^ War ops and Wizards
> 
> Cult, Revs, Traitors, Chang- Literally any antag with a chance of mass death and destruction. Most likely all of them, can cause the major bad moodlet. We need more ways for the major good moodlet

I think I'd be pretty bummed out if I saw mass murder, not sure anything much would cheer me up. For balance reasons it may be worth nerfing the affect negative mood has on speed.

**Comment by Qustinnus** [comment]
negative mood over time gives you slower movement so it doesnt need a nerf, just go do something to get a good mood lol

**Comment by Grazyn** [comment]
I don't know, seems like code to solve a policy issue: non-antags are killing people, escalation policy has failed to stop it, so let's give them in-game debuffs when they kill. I really don't know... maybe it could even work?

I mean, thinking about it, it does fly from a RP perspective: you kill someone, you realize the depravity of your act, you think "what have I done, I'm a monster" and get sad. If you keep killing, eventually you go insane, just like real life

**Comment by Shardj** [comment]
@Grazyn there is no way to have a murderboning policy without going and questioning every decision a syndie makes, they can't kill, but they're valid to everyone else. Justifying every decision, it's a nightmare. Try stations like yog and you'll see the issue, its why the policy discussion going on in the forums at the moment is so against a murderboning policy. That doesn't however mean it shouldn't be discouraged through game mechanics.

**Comment by Chronitonity** [comment]
When will beating people with a bible give them a good moodlet

**Comment by Qustinnus** [comment]
@Chronitonity it already does

**Comment by Chronitonity** [comment]
I love being out of touch

**Comment by Shadowflame909** [comment]
@Shardj In one of the more crisis rounds, like the examples stated here. There's going to be less chance for that good moodlet, if someones murderboning/hull breaches are everywhere.

So unless the task is quite simple and gives you multiple places to carry it out. You're gonna get dabbed on by Lexia 'E-Bow E-Sword' 'Oh Wow, Another Nuke Op Round' '0 CD Fireball' Black.

Well, this was always gonna happen. But it'd sure as hell would make it worse when you lose the ability to run away from them.

**Comment by Cyprex** [comment]
Personally, I'd prefer to see a negative moodlet for personally killing someone instead.

**Comment by Shadowflame909** [comment]
Also how does this prevent murderbone, If most antags that murderbone are immune to the downsides of this?

Seems like it prevents people from killing antags at most. Not the other way around.

Also, holy shit shape change mindswap wiz is about to become harder for the players to deal with.

**Comment by XDTM** [comment]
@Shadowflame909 it's good against casual murder from people who aren't supposed to murder in the first place.

**Comment by coiax** [comment]
I think this PR should be run just as is, and we'll see if people start getting excessively depressed over witnessing death, and thus turn to excessive drinking, smoking, book reading, reassuring hugs, petting corgis, taking long showers, all the things that need to be done in the real world.

**Comment by alexkar598** [comment]
Let’s become lifeweb without the ERP!

**Comment by actioninja** [comment]
This but unironically

**Comment by 4dplanner** [comment]
> the shaftminer main

Lavaland mobs don't have the grief componenet.

> @XDTM
> 
> this pr kills the chef main
> 
> the doctor main
> 
> the shaftminer main
> 
> Any job that involves seeing a lot of things die/corpses in general.
> 
> Oh no no no its just not very compatible with ss13

Some of these don't seem applicable - lavaland mobs don't have the grief component, so shaft miners are fine.

You have to see the death itself to be upset, so bringing corpses to the chef is fine.

Have I misunderstood?

**Comment by optimumtact** [comment]
No you go it, he just didn't

**Comment by Shadowflame909** [comment]
What about antags with dying as their gimmick. Looking at you, lings. I'm unsure about the gibber, but we still butcher Mobs like monkeys with knives.

**Comment by subject217** [comment]
it's time to stop shadowflame

**Comment by Shardj** [comment]
> is there really an epidemic of non-antags going on a murder spree that needs to be addressed with such a radical change?

@Grazyn no, its antags who shouldn't murderbone murderboning that people want to nerf, it reduces the enjoyment for lots of players for no reason. If you agree with that or not is a different thing entirely but you'll see lots of people on the forums feel this way. People don't like random death for no reason, but people also don't like to make policy which restricts too much and is a nightmare to enforce. This is the middleground.

**Comment by XDTM** [comment]
> I'm still iffy about the whole "pushing policy through code" thing but then again, I understand that something must be done when policy is ineffective. I don't think we're at that point though, is there really an epidemic of non-antags going on a murder spree that needs to be addressed with such a radical change?

One particular example is secuirty finding that executing criminals and antagonists is far easier and less risky than imprisoning them. While you can't fault a security player for wanting to absolutely remove a threat, it also denies all potential gameplay involved in trying to escape the permabrig, from sneaking in tools, to using an uplink implants, to getting help from comrades or the AI.

There are other cases, like the already-mentioned validhunters who spend their round gearing up for combat and actively seeking to kill antagonists, at the detriment of people who need them to do their job (e.g. HoP going to the armory and becoming HoS 2.0), and of gimmicky antagonists, who have plans that may be creative but suboptimal and more vulnerable to interference (and here you may argue that they need to git gud, but in this case gud is not necessarily the same as fun), gradually leading to more antags going for the classic and overplayed murderbone tactics to avoid wasting their rare antag round on a plan they have little chance to execute.

I kinda went off on a rant here, and this pr will definitely not solve these issues by itself, but this is why i think it's going in a good direction.

**Comment by Grazyn** [comment]
@Shardj 
>Antag datums now have a mind_traits list var which contains list(TRAIT_GRIM) by default. Some of the more non-violent antagonist datums are equally affected by witnessing death. They are Abductors, non-deathsquad ERT, Valentines, Greentexters and Santa.
This PR hardly addresses it, antags will not get the mood debuff except for those antags stated above, so basically only abductors and ERT (the others are meme/events). I wouldn't even call it a baby step towards what you described, it just has nothing to do with it. And in fact XTDM just explained what he meant for "people who shouldn't murderbone"

**Comment by Shardj** [comment]
@Grazyn my main concern is with greentexters and non antags doing it, neither has mindtrait grim, so yes it does address it. And its great that XDTM is highlighting other advantages, thats why I thumbed up his comment earlier as I agree with it, I'm not sure why you mentioned it.

**Comment by nicbn** [comment]
Hey you are killing the joke.

**Comment by nicbn** [comment]
Have your negative moodlet.

**Comment by nicbn** [comment]
Jokes aside, it would be neat if you made it so that the moodlet stores the name of whoever died, so that every time you see someone else dying you get a different moodlet (the moodlet would probably need a nerf as it will be reapplied a lot of times) and if you ever see that person alive (even if it's not the same person, but has the same name - eg lings or abductors) the moodlet will be removed.

**Comment by BeeSting12** [comment]
Security, medical doctors, the CMO, and roboticist should all be immune to this because they'd be desensitized to it.

**Comment by Grazyn** [comment]
@Shardj Greentexters are those brainwashed by abductors? I thought they were those who pick up the greentext book. In this case I agree with you, abductees shouldn't murderbone but they often do. I referenced XTDM's post because you only mentioned antags, while he explicitly talks about non-antags being affected.

@XDTM So changes like this one, which directly impacts non-antags and doesn't affect antags in the slightest (with a few exceptions) will, by a convoluted 1000 IQ reasoning, eventually lead to a reduction in antag murderboners? I have a hard time believing it.
Let's take a cult round, or any another TDM round. Non-antags like security get a debuff for killing antags, a debuff (slowdown) which impairs their combat ability and makes them easier to kill. Antags don't get such debuff and can kill with no downsides. Yeah this will surely make cultists less likely to murderbone. How is this even remotely close to a step in the right direction?

**Comment by Shardj** [comment]
> Security, medical doctors, the CMO, and roboticist should all be immune to this because they'd be desensitized to it.

@BeeSting12 Security should absolutely not be immune to it, they're to eager to use lethals right away as it is. Doctors should just deal with it, if they're letting their patients die then they should drink to handle the grief. To handle robotocists maybe brain surgery death shouldn't cause grief somehow. Or we go with @nicbn 's sick idea of removing the negative debuff if you see them alive again (including as a borg)

**Comment by Shardj** [comment]
@Grazyn by greentexters I was referring to roles such as regular traitors where they have set objectives and normally have no good reason to murderbone. They use the greentext.dm file. Those which are the exception are specifically marked with the grim trait. However I can see the confusion, I wasn't very clear at all

**Comment by Grazyn** [comment]
@Shardj My knowledge of code is very limited, from the changes in antag_datum.dm I was under the impression that regular traitors were getting the grim trait as well. If this is not the case then ignore what I said, debuffing traitors/ling ecc. for killing is definitely a step in the right direction

**Comment by Shardj** [comment]
greentext.dm overwrites the mind_trait variable for antags which use it no? I'm still learning how the code works for ss13, it's really weird. Either way it needs adjusting to allow and disallow certain roles rather than being a blanket change so I do agree @Grazyn

**Comment by Cyberboss** [comment]
Your PR is getting old. Either Travis CI is failing, it is still labelled work in progress (change this by adding `[READY]` to the start of your title), or there are unaddressed change requests. This may be closed soon to free up space on our tracker if it remains idle. In the event that it is, please ask in #coderbus on Rizon for it to be reopened once you have resumed work on it.

**Comment by coiax** [comment]
I have to remove the antagonist mood boosts, and then I should be ready.

**Comment by optimumtact** [comment]
I wouldn't merge this without a lot more positive moodlets first

**Comment by Grazyn** [comment]
>we need more positive moodlets

Do we really? Book, hug and booze buffs can all be stacked at roundstart, require almost zero effort on your part and will last for quite some time, until food and drugs become available. On top of it, you can get any of the mood-improving traits like joyful, musician (the buff works on the player as well) or neet. Heirloom also gives you a buff as long as you have it IIRC. If you want to go full mr. Brightside, you only need a minimum effort to stack the other buffs (like showering) and you will never know sadness again.

**Comment by coiax** [comment]
Antagonists no longer have mood boosts, they're just grim now.

**Comment by Cyberboss** [comment]
My review is still outstanding

**Comment by optimumtact** [comment]
I wouldn't merge this without a lot more positive moodlets first
--

**Comment by ExcessiveUseOfCobblestone** [comment]
okay

**Comment by DapperMola** [comment]
Roboticists definitely shouldn't need to deal with this, getting a slowdown every time you do your job is Top Tier Unfun. How will Apathetic interact with this?

**Comment by coiax** [comment]
People don't seem to realize that a low mood doesn't cause slowdowns; that a low mood merely lowers your sanity over time, and that eventually leads to a slowdown if you don't take action to boost it.

**Comment by bgobandit** [comment]
I'm not reading 20000 posts but how about some kind of desensitization effect where the first moodlet instance is horrific but the more times it happens the less bad it gets

and then a Desensitized quirk that starts you off at desensitized

**Comment by duckay** [comment]
whats the purpose behind this even

**Comment by tuypo1** [comment]
>whats the purpose behind this even
To remove fun

**Comment by Shadowflame909** [comment]
> 
> 
> whats the purpose behind this even

To slowly turn /tg/ into med-rp

**Comment by ExcessiveUseOfCobblestone** [comment]
> turn /tg/ into med-rp

how far have we fallen?

**Comment by PKPenguin321** [comment]
Why would you be bothered by death in a world with cloning, magic healing wizards, and Lazarus injectors?

**Comment by Mickyan** [comment]
You know, you could just nerf the duration instead of having this sit here for months

There's enough positive moodlets to deal with mood problems, it's just the duration of this one is disproportionate

**Comment by Qustinnus** [comment]
@coiax you should raise the duration of other moodlets

**Comment by coiax** [comment]
I will increase the duration of other moodlets.

**Comment by datorangebottle** [comment]
Doesn't this absolutely fuck over antagonists, robotics, and medical in terms of moodlets? They will likely all be insane by the end of the round. I mean, it's a nerf to murderboning, and punishing that as much as possible is nice, I suppose.

**Comment by 4dplanner** [comment]
Antag datums now have a mind_traits list var which contains list(TRAIT_GRIM) by default. Some of the more non-violent antagonist datums are equally affected by witnessing death. They are Abductors, non-deathsquad ERT, Valentines, Greentexters and Santa.

**Comment by Shadowflame909** [comment]
How does this PR effect you if you have two different antagonist statuses? I've gotten creep as a traitor. Does it mesh well with staff of change antagonists?

**Comment by ExcessiveUseOfCobblestone** [comment]
It checks everything before coming to a conclusion.

**Comment by BeeSting12** [comment]
there should be a "hate someone" trait where you can choose anyone to hate and it gives a positive moodlet if they die but there should also be a love someone trait where you choose someone to love and if they die you get like suicidal level moodlets

**Comment by danzargar** [comment]
Killing ian is a human right

**Comment by coiax** [comment]
I am closing this not because I no longer believe in the idea, but I don't have the time and will at the moment to finish it.

I am VERY happy for someone to continue the PR where I left off.

**Pull request closed without merging**